src/include/distributed/multi_server_executor.h

```diff
@@ -104,6 +104,16 @@ typedef enum
 	CONNECT_ACTION_CLOSED = 2
 } ConnectAction;
 
+/* Enumeration that defines the different log types of cross shard queries */
+typedef enum
+{
+	CROSS_SHARD_INFO_NONE = 0,
+	CROSS_SHARD_INFO_OFF = DEBUG5 - 1, /* TODO : May change it */
```

Comment:
This is hacky. Also, we can ditch `CROSS_SHARD_INFO_NONE ` and have `CROSS_SHARD_INFO_OFF ` as the default one.